ngx-tools/jwt/src/jwt-token-managment/actions.ts

```diff
@@ -36,13 +33,15 @@ export class InitialTokenExtracted implements Action {
   public constructor(public token: string) { }
 }
 
+
```

Comment:
i'm going to have to decline this pr. this newline was deliberately added by pressing the enter key way too hard. i can tell from the whitespace.  please redo this newline with a little more care.

Reply:
![RelievedNeedyBobwhite-size_restricted](https://user-images.githubusercontent.com/270193/61893233-960cc180-aedb-11e9-98d8-217910b5a2d2.gif)